weather_mv/loader_pipeline/bq.py

```diff
@@ -103,28 +108,38 @@ def expand(self, paths):
         """Extract rows of variables from data paths into a BigQuery table."""
         extracted_rows = (
                 paths
-                | 'ExtractRows' >> beam.FlatMap(
+                | 'PrepareCoordinates' >>
+                beam.FlatMap(
+                    prepare_coordinates,
+                    coordinate_chunk_size=self.coordinate_chunk_size,
+                    area=self.area,
+                    open_dataset_kwargs=self.xarray_open_dataset_kwargs,
+                )
+                | beam.Reshuffle()
```

Comment:
@ksic8 I believe that this is the best place for a reshuffle, as we discussed in today's standup.